buildkitd/buildkitd.go

```diff
@@ -42,6 +43,9 @@ var (
 func NewClient(ctx context.Context, console conslogging.ConsoleLogger, image, containerName, installationName string, fe containerutil.ContainerFrontend, earthlyVersion string, settings Settings, opts ...client.ClientOpt) (*client.Client, error) {
 	opts, err := addRequiredOpts(settings, installationName, opts...)
 	if err != nil {
+		if fe.Config().Setting == containerutil.FrontendPodmanShell {
+			return nil, hint.Wrap(err, "podman now requires TLS certs - try re-running 'earthly bootstrap'")
+		}
 		return nil, errors.Wrap(err, "add required client opts")
```

Comment:
```suggestion
		err = errors.Wrap(err, "add required client opts")
		if fe.Config().Setting == containerutil.FrontendPodmanShell {
			return nil, hint.Wrap(err, "podman now requires TLS certs - try re-running 'earthly bootstrap'")
		}
		return nil, err
```

Reply:
thanks, I missed the fact I changed the wrapping.